charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -161,10 +161,12 @@ spec:
 {{- end }}
   podTemplateSpec:
     spec:
+      securityContext: {{- ternary ($defaultSecurityCtx) (.Values.cassandra.podSecurityContext | toYaml) (empty .Values.cassandra.podSecurityContext) | nindent 10 }}
```

Comment:
The template here still pretty redundant and could be extracted into a single helper template that would eliminate a good bit of duplication.